src/content/BatchRequestContent.ts

```diff
@@ -227,14 +227,14 @@ export class BatchRequestContent {
 	 * @returns The Promise that resolves to a body value of a Request
 	 */
 	private static async getRequestBody(request: IsomorphicRequest): Promise<any> {
-		let bodyParsed: boolean = false;
+		let bodyParsed = false;
 		let body;
 		try {
 			const cloneReq = request.clone();
 			body = await cloneReq.json();
 			bodyParsed = true;
 		} catch (e) {
-			// tslint:disable-line: no-empty
+			//TODO- Handle empty catches
```

Comment:
Please create an issue for this and link it here.

Reply:
https://github.com/microsoftgraph/msgraph-sdk-javascript/issues/390